src/constants/translations/en/guest-home-page.json

```diff
@@ -19,7 +56,7 @@
   },
   "whatCanYouDo": {
     "title": "What can you do",
-    "description": "Whether you’re looking for a tutor or looking to become one – you’ve come to the right place.",
+    "description": "Whether you’re looking for a tutor or to looking become one – you’ve come to the right place.",
```

Comment:
I think you shouldn't have changed the order. "looking to" was certainly a correct one